venv_template.mk

```diff
@@ -4,10 +4,10 @@ venv.$(1): venv.$(1)/bin/activate venv.$(1)/bin/environment.sh
 venv.$(1)/bin/activate: venv/$(1)/requirements.txt submodules
```

Comment:
Why not remove submodules if we don't have any?

Reply:
You're right, submodules removed.